### DISCUSSION TITLE
Can'r accces repo

### DISCUSSION THREAD

**Discussion by kalex375** [Laravel-Backpack/community-forum#992] [General] [answered]
I am writing to seek assistance regarding an issue I encountered while trying to update Laravel Backpack from version 4 to version 5. During the update process, I received the following error message:

```
Failed to execute git clone --mirror -- https://ghp...6y2:x-oauth-basic@github.com/Laravel-Backpack/PRO.git C:/Users/Administrator/AppData/Local/Composer/vcs/git-github.com-Laravel-Backpack-PRO.git/
Cloning into bare repository 'C:/Users/Administrator/AppData/Local/Composer/vcs/git-github.com-Laravel-Backpack-PRO.git'...
remote: Repository not found.
fatal: repository 'https://github.com/Laravel-Backpack/PRO.git/' not found
```

It appears that the repository could not be found at the specified URL. I have ensured that my credentials and access permissions are correct.

I followed the upgrade guide provided at [Upgrade Guide](https://backpackforlaravel.com/docs/5.x/upgrade-guide) and added the 
````
composer config http-basic.backpackforlaravel.com 92881900 XXXXXXXXX

````
Our license is registered under the email address development@appfront.cloud.

![image](https://github.com/Laravel-Backpack/community-forum/assets/966329/18c3dd9f-8160-4c69-8147-284ce24dab19)

Could you please assist me in resolving this issue? Any guidance or steps to rectify this problem would be greatly appreciated.

**Comment by karandatwani92** [comment]
Hey @kalex375 

It seems you have solved it yourself🎉.  You can share the solution/understanding with others by answering the discussion.
Incase you are still facing the issue, please let me know.

**Comment by cheadevit** [comment]
I also face this problem, please help.

**Reply by kalex375** [reply, marked as answer]
You need set exact version from backpack dashboard in your composer.json.

**Reply by pxpm** [reply]
Hey @cheadevit, like @kalex375 said (thank you 🙏 ) you need to pin the version in your composer you last have access to. 

My guess is that you have in your `composer.json` `"backpack/pro": "^2.0"` and you only have access to some 1.xx version. If that's the case just use `backpack/pro: ^1.0` in your `composer.json` and it should work. 

If not, please provide some more details on a new discussion, as this one is already closed and we don't need to bother @kalex375 that already solved his issue 👍 

Cheers
